### DISCUSSION TITLE
Using Ensemble-Stat output (NMEPs) in Grid-Stat

### DISCUSSION THREAD

**Discussion by spence98** [dtcenter/METplus#1805] [Statistical Computation] [answered]
Hello MET help!

I am wondering if you could help me with figuring out how to compute the FSS using the Grid-Stat tool, but using Ensemble-Stat output (NMEPs) to compute the FSS. I found this: 

“The Ensemble-Stat tool writes the gridded relative frequencies, NEP, and NMEP fields to a NetCDF output file. Probabilistic verification methods can then be applied to those fields by evaluating them with the Grid-Stat and/or Point-Stat tools.”

But I haven't been able to find anything in the documentation or the use cases that shows how to do this process. 

Thank you for your help!
Michelle

**Comment by CPKalb** [comment]
Hi Michelle:

I'm guessing what you are asking is how to set up the FSS, since the neighborhoods are already computed in the NMEP fields?  When using probabilistic output from Grid-Stat, this issue doesn't arise, so it's simple to compute the probabilistic statistics.  I actually don't know the answer to this and need to touch base with some others on the MET team.  I'll do that and get back to you.

If that is not what you are asking, then I apologize for getting it wrong.  In this case could you give more specifics?

Thanks,
Christina

**Comment by CPKalb** [comment]
Hi Michelle:

After discussion with @JohnHalleyGotway, it sounds like you would need to change the field value in the neighborhood (nbrhd) dictionary to be OBS, rather than BOTH.  With these settings, MET will compute a neighborhood on the observation data but leave the forecast as it is.   Here, you would want to make sure that the neighborhood logic applied to the observation is the same as how the forecast values were defined (i.e. same size and shape).  It might be helpful to define the "nbrhd" dictionary separately for each field being verified, which would require putting the neighborhood dictionary inside the variable field dictionaries. 

I am checking to see if we happen to have any examples of this that could be used to help with the setup.

Christina

**Reply by spence98** [reply]
Hello Christina,

Thank you for your reply, although I don't think it fully addresses my situation, which I don't think I've made clear enough. 

The way that I am planning to calculate the FSS is using solely NMEP results. I'm following an approach that consists of using a single forecast and single observation dataset to create the NMEP (basically my 'ensemble' is one forecast and one observation). I then want to use the NMEP output as my input into gridstat to calculate the FSS. So, I wouldn't be using the BOTH  or the OBS as the field value in the neighborhood dictionary. Is NEITHER and option for that dictionary? 

I hope this all makes sense. My overall goal is to obtain an FSS value for each of my events, using each forecast against the LSRs for that event. I originally tried this method using only gridstat tools, but after some further lit review, I think that utilizing the neighborhood maximum ensemble probabilities would lead to better results.

Thank you!
Michelle

**Comment by CPKalb** [comment, marked as answer]
Hi Michelle:

Thanks for clarifying. The field in the nbrhd dictionary can be set to NONE. Using this setting, MET will assume that fractional coverages (values between 0 and 1) were already computed outside of Grid-Stat.

Christina

**Comment by CPKalb** [comment]
Hi Michelle:

I'm following up to check if you able to get this working?  Does the NONE option work?

Thanks,
Christina

**Comment by CPKalb** [comment]
I am going to mark this discussion as answered and lock it.  If you have additional questions or issues, feel free to start up a new discussion.

Thanks,
Christina
